Documentation: Contributing system requirements

Hi.

Thank you for dotliquid.

When I open the DotLiquid.sln in Visual Studio 2015 I get the following error message:
```
---------------------------
Microsoft Visual Studio
---------------------------
The following error occurred attempting to run the project model server process (1.0.1). 
Unable to start the process. No executable found matching command "dotnet-projectmodel-server"
The project model server process provides intellisense, build, and reference information to Visual Studio and without it your experience will be very limited. Please try closing and reopening Visual Studio to see if that corrects the problem. Alternatively, check that the .NET Core SDK is properly installed.
---------------------------
OK   
---------------------------
```

Trying to build the solution anyway produces the following error message in all four projects:
```
*.xproj(7,3): error MSB4019: The imported project "C:\Program Files\dotnet\sdk\1.0.1\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
```

Windows 7 x64 SP1
Visual Studio 2015 Enterprise Update 3
.NET Core SDK 1.0.1 (x64)
Microsoft .NET Core 1.0.1 - SDK 1.0.0 Preview 2-0031313 (x64)
Microsoft .NET Core 1.0.1 - VS 2015 Tooling Preview 2

Any ideas what I'm missing?

Thank you.

--- a/contributing.md
+++ b/contributing.md
@@ -1,0 +1,31 @@
+## Getting started
+
+**Getting started with Git and GitHub**
+
+ * [Setting up Git for Windows and connecting to GitHub](http://help.github.com/win-set-up-git/)
+ * [Forking a GitHub repository](http://help.github.com/fork-a-repo/)
+ * [The simple guide to GIT guide](http://rogerdudler.github.com/git-guide/)
+
+Once you're familiar with Git and GitHub, clone the repository.  
+The requirements are [Visual Studio 2017](https://www.visualstudio.com/) and [.net Core 1.0.3](https://www.microsoft.com/net/download/core#/sdk).
+
+## Discussing ideas 
+
+* [Gitter Chatroom](https://gitter.im/dotliquid/dotliquid)
+* [GitHub Issues](https://github.com/dotliquid/dotliquid/issues/new)
+
+**The functionality is based as much as possible on existing functionality in Liquid**
+
+Each new feature must have been discussed first before submitting it in a PR.
+
+## Coding conventions
+
+We prefer spaces over tabs for indentation.  
+We have an [editorconfig](http://EditorConfig.org) [file](./.editorconfig) if you use an editor or plugin respecting it.
+
+## Testing
+
+Tests are mandatory for new functionality, please add some in the tests suite.
+You can see the result either:
+- in Visual Studio or other IDE supporting NUnit 3
+- on [AppVeyor](https://ci.appveyor.com/project/tgjones/dotliquid) after the PR is submitted

--- a/contributing.md
+++ b/contributing.md
@@ -14,7 +14,7 @@ The requirements are [Visual Studio 2017](https://www.visualstudio.com/) and [.n
 * [Gitter Chatroom](https://gitter.im/dotliquid/dotliquid)
 * [GitHub Issues](https://github.com/dotliquid/dotliquid/issues/new)
 
-**The functionality is based as much as possible on existing functionality in Liquid**
+**The functionality is based as much as possible on existing functionality in [Liquid](https://shopify.github.io/liquid/)**
 
 Each new feature must have been discussed first before submitting it in a PR.
 

--- a/issue_template.md
+++ b/issue_template.md
@@ -1,0 +1,7 @@
+### Dotliquid version
+
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce the Problem (you can add files)

